Docker/validate_sc2.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+"""Validate SC2."""
+
+import argparse
+import json
+
+import dendropy
+
+
+def valid_tree_size(tree):
+    """Check size of prediction tree."""
+
+    return len(tree.taxon_namespace) == 1000
```

Comment:
can parametrize the length

Reply:
Agreed.  Should I maybe create a separate module for this?